java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -48,4 +48,44 @@ public interface StringCommands {
      *     is set, return the old value as a <code>String</code>.
      */
     CompletableFuture<String> set(String key, String value, SetOptions options);
+
+    /**
+     * Increments the number stored at <code>key</code> by one. If <code>key</code> does not exist, it
+     * is set to 0 before performing the operation.
+     *
+     * @see <a href="https://redis.io/commands/incr/">redis.io</a> for details.
+     * @param key The key to increment its value.
+     * @return The value of <code>key</code> after the increment. An error is raised if <code>key
+     *     </code> contains a value of the wrong type or contains a string that can not be represented
+     *     as integer.
+     */
+    CompletableFuture<Long> incr(String key);
+
+    /**
+     * Increments the number stored at <code>key</code> by <code>amount</code>. If <code>key</code>
+     * does not exist, it is set to 0 before performing the operation.
+     *
+     * @see <a href="https://redis.io/commands/incrby/">redis.io</a> for details.
+     * @param key The key to increment its value.
+     * @param amount The amount to increment.
+     * @return The value of <code>key</code> after the increment. An error is raised if <code>key
```

Comment:
Do you want to correct the period in the Node client too?
* https://github.com/aws/glide-for-redis/blob/main/node/src/BaseClient.ts#L456
* https://github.com/aws/glide-for-redis/blob/main/node/src/BaseClient.ts#L468
* https://github.com/aws/glide-for-redis/blob/main/node/src/Transaction.ts#L206
* https://github.com/aws/glide-for-redis/blob/main/node/src/Transaction.ts#L219